xcube_geodb/core/geodb.py

```diff
@@ -1525,10 +1531,9 @@ def head_collection(self, collection: str, num_lines: int = 10, database: Option
 
         return self.get_collection(collection=collection, query=f'limit={num_lines}', database=database)
 
-    def get_collection(self, collection: str, query: Optional[str] = None, database: Optional[str] = None,
-                       limit: int = None, offset: int = None) -> Union[GeoDataFrame, DataFrame]:
+    def get_collection(self, collection: str, query: Optional[str] = None, database: Optional[str] = None) -> Union[GeoDataFrame, DataFrame]:
```

Comment:
This is not up to date, right? Because `limit` and `offset` are now available with get collection, if I remember correctly.

Reply:
Yes, this is an old PR and has been overtaken by others. I will update it and re-request your review.